README.md

```diff
@@ -51,7 +52,7 @@ Once you've joined Operation Code as a member, after your first PR (pull request
 
 ## Open Source Maintenance Policy
 
-You can find our official [Open Source Maintenance Policy](open_source_maintenance_policy.md) in this repo.  
+You can find our official [Open Source Maintenance Policy](open_source_maintenance_policy.md) in this repo.
```

Comment:
This looks good, but it's mixed in with the other stand alone projects. This should be higher at the top as it's not a stand alone project. It might benefit from a different styling instead of the `##`